fixtures/tx_index_schema.sql

```diff
@@ -120,29 +121,3 @@ WHERE tr.key = 'recipient'
   AND tr.type = 'transfer'
   AND ts.type = 'transfer'
   AND tv.type = 'transfer';
-
-
-CREATE MATERIALIZED VIEW cosmos_osmosis_ibc_transfers AS
-  SELECT
-    data.chain_id,
-    data.event_id,
-    cast(data.value AS json) ->> 'receiver' as recipient,
-    cast(data.value AS json) ->> 'sender' as sender,
-    cast(data.value AS json) ->> 'amount' || 'uatom' as amount,
-    data.created_at
-  FROM tx_events data
-  JOIN tx_events source
-    ON source.event_id = data.event_id
-  JOIN tx_events destination
-    ON destination.event_id = data.event_id
-  WHERE (data.composite_key = 'send_packet.packet_data'
-        OR data.composite_key = 'recv_packet.packet_data')
-    AND ((source.composite_key = 'send_packet.packet_src_channel'
-          AND source.value = 'channel-141')
-        OR (source.composite_key = 'recv_packet.packet_src_channel'
-          AND source.value = 'channel-0'))
-    AND ((destination.composite_key = 'send_packet.packet_dst_channel'
-          AND destination.value = 'channel-0')
-        OR (destination.composite_key = 'recv_packet.packet_dst_channel'
-          AND destination.value = 'channel-141'))
-    AND data.chain_id = 'cosmoshub-4';
```

Comment:
This was specific to an experiment I was conducting, and it accidentally slipped in.